data/json/items/ammo.json

```diff
@@ -720,19 +720,15 @@
         "ammo_type" : "NULL",
         "flags" : ["UNRECOVERABLE"]
     },
-    {   "type" : "AMMO",
+    {   "type" : "GENERIC",
         "id" : "5x50_hull",
-        "category" : "spare_parts",
-        "price" : 80000, "//" : "Should be $4 per.",
+        "copy-from" : "casing",
```

Comment:
Won't this create brass casing when this should really be more similar to a shotgun hull?